Documentation/CameraSystem/CameraSystemOverview.md

```diff
@@ -0,0 +1,69 @@
+# Camera System
+
+The camera system enables the Microsoft Mixed Reality Toolkit to configure and optimize the application's camera for use in mixed reality applications. Using the camera system, applications can be written to support both opaque (ex: virtual reality) and transparent (ex: Microsoft HoloLens) devices without needing to write code to distinguish between, and accommodate for, each type of display.
+
+## Enabling the camera system
+
+The Camera System is managed by the MixedRealityToolkit object (or another service registrar component).
+
+The following steps presume use of the MixedRealityToolkit object. Steps required for other service registrars may be different.
+
+1. Select the MixedRealityToolkit object in the scene hierarchy.
+
+![MRTK Configured Scene Hierarchy](../Images/MRTK_ConfiguredHierarchy.png)
```

Comment:
I think if you add a few spaces (3 or 4?) at the beginning here, these will appear more in-line with the list

Reply:
i'll give that a try

Reply:
that looks much better. THANKS!